app/bundles/CoreBundle/Assets/css/libraries/builder.less

```diff
@@ -203,4 +203,7 @@ div[data-slot].ui-sortable-helper {
     .select-theme-link {
         margin-top: 5px;
     }
+    .select-theme-selected {
+        margin-top:5px;
+    }
```

Comment:
It works well! Althouth this PR also changes the permission of the file `100644 → 100755`. The permission should not be changed. And when we're in it, the code could be a one line change if the line 203 would change to `.select-theme-link, .select-theme-selected {`, but that's a minor optimization.

Warning for other testers: The git pull command did not pull the changes:
```
$ git pull https://github.com/RCheesley/mautic.git #2963-button-jumps-when-theme-selected
From https://github.com/RCheesley/mautic
 * branch                HEAD       -> FETCH_HEAD
Already up-to-date.
```
I had to copy-paste the change manually and run the `grunt compile-less` command to see the change. Maybe because there is the `#` in the branch name or something like that.